extensions/datahub/.tool-versions

```diff
@@ -0,0 +1,2 @@
+yarn 1.22.22
+nodejs 23.9.0
```

Comment:
What is this file for?

Reply:
The version manager asdf uses it to tell what versions to use for each tool locally